app/renderer/js/components/webview.ts

```diff
@@ -1,30 +1,38 @@
 'use strict';
 
-const path = require('path');
-const fs = require('fs');
+import * as path from 'path';
+import * as fs from 'fs';
+import { remote } from 'electron';
 
-const ConfigUtil = require(__dirname + '/../utils/config-util.js');
-const SystemUtil = require(__dirname + '/../utils/system-util.js');
-const { app, dialog } = require('electron').remote;
+import ConfigUtil = require('../utils/config-util');
+import SystemUtil = require('../utils/system-util');
+import BaseComponent = require('../components/base');
+import handleExternalLink = require('../components/handle-external-link');
 
-const BaseComponent = require(__dirname + '/../components/base.js');
-const handleExternalLink = require(__dirname + '/../components/handle-external-link.js');
+const { app, dialog } = remote;
 
 const shouldSilentWebview = ConfigUtil.getConfigItem('silent');
+
 class WebView extends BaseComponent {
-	constructor(props) {
+	props: any;
+	zoomFactor: number;
+	badgeCount: number;
+	loading: boolean;
+	customCSS: string;
+	$webviewsContainer: any ;
```

Comment:
Hmm, shouldn't this be `Element` or `HTMLElement`?

Reply:
Should be ```DOMTokenList```